docs/src/user/user-intro.adoc

```diff
@@ -32,7 +32,7 @@ complete system:
 The below illustration is a simple block diagram showing what a typical 3-axis, CNC mill with stepper
 motors might look like:
 
-.Simple LinuxCNC Controlled Machine
+//.Simple LinuxCNC Controlled Machine
```

Comment:
special reason for this and following?

Reply:
Limitation by po4a - the same text appearing in different contexts and this thing freaks out. I presume there is an image::images line beneath.

Reply:
Oh ok. Yes there is an image below.  I just wonder as we are using image captions at at some more locations..

Reply:
We need a style guide. ;) I am not exactly sure about what the difference is between these image captions and the first argument given to the image[] or how this relates to the alt="" argument. These comments are dead easy to undo, so I am not worried.
You will also see that I converted all alt="" to a first argument setting.

Reply:
I think
`image::images/icons/home.png[Alt text]` 
should be the same as
`image::images/icons/home.png[alt="Alt text"]`
but completely different from
```
.Image caption
image::images/icons/home.png[]
```

Reply:
That is what my intuition also told me. You will find though that in 99.9% of all cases in the LinuxCNC docs the first argument is identical to the first argument/the alt attribute. A majority of image calls has no caption at all and only the first argument/alt attribute.

Reply:
>Limitation by po4a - the same text appearing in different contexts and this thing freaks out. 

You mean po4a can't handle it if two equal source strings come from different contexts?